pkg/gardenlet/apis/config/validation/validation.go

```diff
@@ -74,6 +74,17 @@ func ValidateGardenletConfiguration(cfg *config.GardenletConfiguration, fldPath
 		}
 	}
 
+	exposureClassHandlersPath := fldPath.Child("exposureClassHandlers")
+	for i, handler := range cfg.ExposureClassHandlers {
+		handlerPath := exposureClassHandlersPath.Index(i)
+		if len(handler.Name) == 0 {
+			allErrs = append(allErrs, field.Invalid(handlerPath.Child("name"), handler.Name, "handler name cannot be empty"))
+		}
+		if len(handler.LoadBalancerService.Annotations) == 0 {
```

Comment:
OK, now I'm confused :) If it cannot be empty then above questions don't make sense anymore. You should rather just remove the 'optional' tag of this `Annotations` field.

Reply:
see above. done